dotCMS/src/main/java/com/dotcms/ai/app/AppConfig.java

```diff
@@ -11,8 +11,14 @@
 import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 
+/**
+ * The AppConfig class provides a configuration for the AI application.
+ * It includes methods for retrieving configuration values based on given keys.
+ */
 public class AppConfig implements Serializable {
```

Comment:
Same here. This could've been a separate PR for including missing javadoc.

Reply:
Extracted this into a commit and removed from current branch.